lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -886,10 +897,9 @@ ChangeResult TypeAnalyzer::visitOperation(
     Type firstResDtype = operands[0]->getValue().dtype;
     Type secondResDtype =
         IntegerType::get(op->getContext(), 64, IntegerType::Signed);
-    ChangeResult firstRes = visitReductionAlongDimIntOp(
-        maxDim, maxDim.dim(), maxDim.keepdim(), firstResDtype, operands);
-    return firstRes |
-           visitReductionAlongDimIntOp(maxDim, maxDim.dim(), maxDim.keepdim(),
+    visitReductionAlongDimIntOp(maxDim, maxDim.dim(), maxDim.keepdim(),
+                                firstResDtype, operands);
+    return visitReductionAlongDimIntOp(maxDim, maxDim.dim(), maxDim.keepdim(),
```

Comment:
Nit: I'd to the same thing here as up above for consistency and add the `return` on a newline.